_stbt/core.py

```diff
@@ -689,6 +689,18 @@ def __repr__(self):
         return str(self)
 
 
+class OcrPatterns(object):
+    """
+    Common tesseract_user_patterns arguments you might want to pass when calling
+    ocr.
+
+    >>> import stbt
+    >>> stbt.ocr(frame=load_image("tests/ip_address.png"))
+    u'192.168.1.64'
+    """
+    IP_ADDRESS = (r"\d\*.\d\*.\d\*.\d\*",)
```

Comment:
If this was just a string (not a tuple with 1 string) it would be easier to combine with other patterns, e.g. 

    tesseract_user_patterns=[OcrPatterns.IP_ADDRESS, OcrPatterns.SOMETHING_ELSE]

Since #474 it's safe to pass a single string if you only have 1 pattern.